server/app/assets/javascripts/map.ts

```diff
@@ -0,0 +1,48 @@
+import {Map} from 'maplibre-gl'
+
+export const init = () => {
+  const geoJsonDataObject = window.app.data.maps || {}
+  const mapElements = document.querySelectorAll('[id^="cf-map-"]')
```

Comment:
You don't need this anymore. You can just look over the items in `window.app.data.maps`